packages/app/src/containers/LayerThemeInspector.tsx

```diff
@@ -43,50 +118,54 @@ export default memo(function LayerThemeInspector() {
         <InspectorPrimitives.Row>
           <Select
             id="theme-layer-style"
-            value={'No Layer Style'}
+            value={selectedLayerStyleId || 'No Layer Style'}
```

Comment:
I know I wrote this, but can we use the constant here, `selectedLayerStyleId || NO_LAYER_STYLE`?